data/compatibility/supportStatus.yml

```
@@ -5,13 +5,14 @@
   supported: "No, development only"
   releaseDate:
   eolDate:
-  k8sVersions: ["1.25", "1.26", "1.27", "1.28"]
+  k8sVersions: ["1.25", "1.26", "1.27", "1.28", "1.29"]
   testedK8sVersions: ["1.23", "1.24"]
```

Based on my prior remark
```suggestion
  k8sVersions: ["1.26", "1.27", "1.28", "1.29"]
  testedK8sVersions: ["1.23", "1.24", "1.25"]
```

---

Ack - fixed